i18n/en.json

```diff
@@ -1114,6 +1114,10 @@
     "id": "api.config.client.old_format.app_error",
     "translation": "New format for the client configuration is not supported yet. Please specify format=old in the query string."
   },
+  {
+    "id": "api.config.update_config.clear_siteurl.app_error",
+    "translation": "Can't clear Site URL."
```

Comment:
@ashishbhate Can we update this to "Site URL cannot be cleared."

Reply:
@wiersgallak done